README.md

```diff
@@ -137,6 +139,14 @@ Add the CSS and JavaScript code to your HTML template:
 </html>
 ```
 
+### What is different in the `old-ie` stylesheet?
+
+The difference between `govuk-frontend.min.css` and `govuk-frontend-old-ie.min.css`
```

Comment:
Possibly an oversimplification – as well as rasterising media queries we also have IE8 specific fixes throughout the codebase, and do things like convert rgba to rgb.

This also only helps those using the `dist` versions – we also need to tell the user how to generate both versions from their own app stylesheet.

Reply:
I'm not sure we need to go into detail, all people really need to know is that this file contains additional CSS for IE8 support.

Reply:
Yep, I think I'd agree, but I think this is slightly misleading in that it suggests that the _only_ difference is media query rasterising.

Reply:
I think just stating that IE8 need a different build for the CSS would be fair enough. Otherwise we'll end up updating these documented differences each time we update the CSS that targets IE8.

Reply:
I think @alex-ju is right, this makes the file more maintainable. I've simplified the explanation.